ledger/participant-integration-api/src/main/scala/platform/index/IndexServiceOwner.scala

```diff
@@ -38,8 +40,7 @@ final class IndexServiceOwner(
     participantId: Ref.ParticipantId,
     inMemoryState: InMemoryState,
 )(implicit
-    loggingContext: LoggingContext,
-    executionContext: ExecutionContext,
```

Comment:
Not needed since it was only used for initialization of resources (which should be covered by the `def acquire()(implicit context: ResourceContext)`